relay-server/src/endpoints/common.rs

```diff
@@ -356,6 +332,9 @@ pub async fn handle_envelope(
             envelope,
             state.outcome_aggregator().clone(),
             state.test_store().clone(),
+            // It's not clear at this point which group this envelope belongs to.
+            // The decission will be made while queueing in `queue_envelope` function.
+            ProcessingGroup::Unknown,
```

Comment:
Ideally this would be reflected in the type somehow (for example `ManagedEnvelope` vs. `GroupedManagedEnvelope`), but that would be too big of a refactor now, so this is fine.